### PR TITLE
Map with point and text

### PR DESCRIPTION
**Pull request by domoritz**
Depends on #3515

### CODE REVIEW

**Comment by kanitw** [comment]
Now doable with https://github.com/vega/vega-lite/pull/3515?

**Comment by domoritz** [comment]
Yes

image

File: examples/specs/geo_text.vl.json [hunk 1]
```diff
@@ -36,9 +32,22 @@
         "y": {
           "field": "lat",
           "type": "latitude"
+        }
+      },
+      "layer": [{
+        "mark": "circle",
```
Comment (kanitw) [inline examples/specs/geo_text.vl.json]: Why not using color in mark?  (more concise)

Comment (domoritz) [reply examples/specs/geo_text.vl.json]: Will do when I update the spec for you lat/long PR.

**Pull request merged**